examples/model_assisted_labeling/debugging_mal.ipynb

```diff
@@ -153,7 +153,7 @@
    "source": [
     "project = client.create_project(name=\"debugging_mal_project\")\n",
     "dataset = client.create_dataset(name=\"debugging_mal_dataset\")\n",
-    "test_img_url = \"https://upload.wikimedia.org/wikipedia/commons/thumb/0/08/Kitano_Street_Kobe01s5s4110.jpg/2560px-Kitano_Street_Kobe01s5s4110.jpg\"\n",
+    "test_img_url = \"https://raw.githubusercontent.com/Labelbox/labelbox-python/sdk/examples/assets/2560px-Kitano_Street_Kobe01s5s4110.jpg\"\n",
```

Comment:
I would change these urls to the develop branch instead of sdk.
```raw.githubusercontent.com/Labelbox/labelbox-python/sdk/examples/assets```
should be 
```raw.githubusercontent.com/Labelbox/labelbox-python/develop/examples/assets```

Reply:
(for all links to this asset)